src/System.Linq.Expressions/tests/SequenceTests/SequenceTests.cs

```diff
@@ -4266,6 +4266,31 @@ public static void TestConvertToNullable()
             Assert.Equal(2, d(2));
         }
 
+        [Fact]
```

Comment:
Remind me: will these tests run with both the compiled and interpreted versions?